inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -179,11 +156,21 @@ void FrontEnd::detectNetworkBatch(
     env.log->trace("Reshape the network");
 
     ie::ResponseDesc desc;
-    const auto status = network.reshape(inputShapes, &desc);
+    auto status = ie::StatusCode::OK;
+    try {
+        status = network.reshape(inputShapes, &desc);
+    }
+    catch (...) {
+        status = ie::StatusCode::GENERAL_ERROR;
+    }
 
-    VPU_THROW_UNLESS(
-        status == ie::StatusCode::OK,
-        "Failed to reshape Network: %v", desc.msg);
+    if (status != ie::StatusCode::OK) {
+        env.log->trace("Failed to reshape Network: %v", desc.msg);
+        return;
+    }
```

Comment:
Do we expect that we're going to be able to infer network if there were a batch but we failed to extract it?

Reply:
yes, we do )